ads/google/ima-player-data.js

```diff
@@ -23,7 +23,7 @@ export class ImaPlayerData {
     this.currentTime = 0;
 
     /* {!Number} */
-    this.duration = 1;
+    this.duration = NaN;
```

Comment:
Since other players just return `1` to mean unsupported, let's revert this to `1` and change the condition on the percentage tracker to exclude `<= 1` as discussed offline.